consensus/consensus.md

```diff
@@ -280,8 +288,8 @@ EDGEBITS: 28
 
 
 ```
-header = version || height || prev_hash || txs_hash || root_hash || target || evidence || nonce || time
-           64         64         256         256          256         64       42*32       64       64
+header = version || height || prev_hash || root_hash || target || evidence || nonce || time
```

Comment:
Miner and beneficiary? ([code](https://github.com/aeternity/epoch/blob/f3676d1d44503cbd3c94c5363f4d37cb04f48cc4/apps/aecore/src/aec_conductor.erl#L320))

See also https://github.com/aeternity/protocol/blame/f0f6c18ee27fd1f349f613c3bb3dbac7034f7ec3/serializations.md#L35-L36